Join us as a member (#22)
publiccodenet/http: Founding-membership.md

# Become a founding member

We’re currently looking for founding members to help build the Foundation for Public Code’s long term future by joining us now.

Founding members are forward-thinking public organisations with a strong principled commitment to the existence of an ecosystem of public code. You’ll be willing to take co-ownership of the Foundation [in line with our draft principles] and be an advocate for public code.

We are looking for organisations that can either:

* help us validate our code certification and stewardship processes
* provide material support to create a public code ecosystem

We’ll open other types of membership from 2020.

## Benefits of being a founding member

As a founding member, you’ll shape the Foundation’s processes and governance.

As a founding member, you will:

> * be one of the first to submit your codebases for stewardship by the Foundation (reducing your future risk and maintenance costs)

Reviewer: ```suggestion
* be one of the first to have codebases stewarded by the Foundation (reducing your future risk and maintenance costs)
```

I find 'submit' to be a bit of a harsh word, it also seems to be like a process that doesn't involve them.

* get a seat on the board of directors

> * have your membership fees waived for the first year

Reviewer: I don't think this should be a standard offer, since we're going to have to ask Founding members to fund us, often the first year already since we don't have money until the end of 2020.

Author: I've removed this.

* steer the Foundation’s pipeline towards projects you’d like to reuse
* gain recognition as a leader of systemic digital transformation

This is in addition to the regular benefits of Foundation membership. These include being able to:

* invoke continuous integration auditing resources on projects in development accepted by the Foundation (increasing the security of your code)

> * (re)use codebases from the repository, and rely on the stewarded community for support, bug fixes, and continued development (saving money on future procurement)

Reviewer: Anyone gets this benefit, perhaps we can put something in there about there privileged role

* demonstrate return on IT investments from higher levels of administration
* learn from and share experience with other bold, innovative public organisations
* collectively participate in a trusted Foundation procurement strategy

We also offer:

* access to Foundation leaders and advisors for open source strategy discussions
* entry to exclusive, invite-only public code leadership summits focused on technical, compliance, and business topics
* prominent placement of your organization’s brand on the Foundation’s social networks and web properties
* direct access to technical decision-makers of the Foundation platform
* opportunities for leadership roles in Foundation initiatives
* a blogpost announcing membership

> 

Reviewer: Extra white rule

> ## Founding membership: process validation

Reviewer: I still think splitting these up into different kinds of membership is more confusing than enlighting.

Perhaps we can split them not as kinds of membership but as two asks of members, they can provide one, the other or both.

Author: This makes it harder for public orgs to self-identify as not qualifying for founding membership. Collapsing the two categories also makes it harder for orgs without available code to see that there's also a route for them.

```
## Founding membership requirements

To become a founding member you should have some or all of these:

* an open source software solution that you think other public organisations will be interested in reusing
* a development team and policy experts able to work with the Foundation to refactor and document your software and policy so that they meet the Standard for Public Code
* enthusiasm for public code at the most senior levels within your organisation
* a willingness to help the Foundation test its onboarding and code auditing processes, so we can get it right
* the ability to make in-kind contributions to help us develop our processes and governance, so that they work better for your type of organisation
* the ability to make a significant financial commitment relative to your type and size of organization (especially if you don’t currently have a codebase available, but are invested in creating a future for public code)
```
